CRM/Civicase/Service/CaseCategoryMenu.php

```diff
@@ -68,36 +68,36 @@ private function createCaseCategorySubmenus($caseTypeCategoryName, $caseTypeCate
       [
         'label' => ts('Dashboard'),
         'name' => "{$caseTypeCategoryName}_dashboard",
-        'url' => "civicrm/case/a/?ctc={$caseTypeCategoryId}#/case?ctc={$caseTypeCategoryId}",
+        'url' => "civicrm/case/a/?p=dd&ctc={$caseTypeCategoryId}#/case?ctc={$caseTypeCategoryId}",
```

Comment:
Even though it is a good idea to add these parameters for identifying the pages, eventually they will be scattered all over the place and it will be difficult to even know what they mean or what each abbreviations mean.

Will be good to have this defined somewhere in a class or function that the Breadcrumb function will use for replacement and we can always refer to the function for more details about these abbreviations.

A simple switch or if statement in a `CaseCategoryPage` class will suffice. 

e.g 
```php

  public function getPageDetails($abbreviation) {
    if ($abbreviation = 'pd') {
      return ['name' => 'Page Dashboard', 'details' => 'Page Dashboard Page'];
    }
  }

```

NB. This can be done on the breadcrumb PR.